packages/create-cloudflare/e2e-tests/cli.test.ts

```diff
@@ -71,11 +48,12 @@ describe.skipIf(experimental || frameworkToTest || isQuarantineMode())(
 			expect(output).toEqual(version);
 		});
 
-		test.skipIf(process.platform === "win32")(
+		test({ experimental }).skipIf(process.platform === "win32")(
 			"Using arrow keys + enter",
-			async () => {
+			async ({ logStream, project }) => {
+				console.log(project);
```

Comment:
Noticed this pop up the the output https://github.com/cloudflare/workers-sdk/actions/runs/11072563792/job/30767063710?pr=6216#step:4:190

Was it intentional?

Reminds me: can we log in the job output what pm+framework failed so we can more easily distinguish flakes from repeated real failures without opening the logs file?

Reply:
Again hacking/debugging leftovers.

But yeah I agree we should make the error cases more easy to diagnose.